api/src/wfl/module/covid.clj

```diff
@@ -34,7 +79,7 @@
     (if (and started (not finished)) (update! workload) workload)))
 
 (defoverload workloads/create-workload!   pipeline create-covid-workload!)
-(defoverload workloads/start-workload!    pipeline start-covid-workload!)
+(defoverload workloads/start-workload!    pipeline batch/stop-workload!)
```

Comment:
This looks like it is incorrect.

Reply:
yeah, this needs to be updated when we implement the `start` for covid anyways.

Reply:
FWIW: I have a mock on my branch.